hooks/playbooks/ceph.yml

```diff
@@ -0,0 +1,526 @@
+---
```

Comment:
This is a huge new file. Did you `git mv` it from the previous location?
I don't know if that's github UI, but it looks like a brand new file.

Reply:
yes, mv to another place, but later I created a symlink, so it shows as it shows.